src/main/java/com/robin/hoteless/Reservation.java

```diff
@@ -0,0 +1,42 @@
+package com.robin.hoteless;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    int id;
+    Date made;
+    int stayDays;
+    double dailyRate;
+    LocalDate in;
+    LocalDate out;
+    String bedType;
+    double total;
+    static int totalReservations;
+
+    Reservation(Date m, int s, LocalDate i, LocalDate o, double d, String b, double t) {
+        id = totalReservations + 10000;
+        made = m;
+        stayDays = s;
+        in = i;
+        out = o;
+        dailyRate = d;
+        bedType = b;
+        total = t;
+        totalReservations++;
+    }
+
+    void show() {
+        System.out.printf(
```

Comment:
You'll want to figure out how to set up a logger to output data, since ``System.out.println()`` is for debugging purposes only. :)